server/services/mapping_service.py

```diff
@@ -1,3 +1,4 @@
+import sys
```

Comment:
I'm not seeing `sys` used anywhere else in the file.

Reply:
This was from a commit that wasn't supposed to be included. Fixed